src/transactions/feed/TransactionFeedV2.test.tsx

```diff
@@ -424,11 +423,15 @@ describe('TransactionFeedV2', () => {
 
     // imitate changing of pending stand by transaction to confirmed
     await act(() => {
-      const changePendingToConfirmed = transactionConfirmed(
-        standByTransactionHash,
-        { status: TransactionStatus.Complete, transactionHash: standByTransactionHash, block: '' },
-        mockTransaction().timestamp
-      ) as Action
+      const changePendingToConfirmed = transactionConfirmed({
+        txId: standByTransactionHash,
```

Comment:
Unrelated to this PR, but is this right? txId is not the TX hash.

Reply:
@jeanregisser `txId` is used to compare the value from `context.id` of a stand by transaction. For the sake of testing the actual value of ID is irrelevant as it just should match the ID of the same transaction between pending and confirmed state. At least, that's my observation from some existing usages, like [here](https://github.com/valora-inc/wallet/blob/main/src/jumpstart/saga.test.ts#L249).